core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1966,6 +1969,16 @@ private[spark] object Utils extends Logging {
     getHadoopFileSystem(new URI(path), conf)
   }
 
+  /**
+   * Return a new `ObjectMapper` with `ClassTagExtensions`.
+   * The mapper registers `DefaultScalaModule` by default.
+   */
+  def withScalaModuleMapper: ObjectMapper with ClassTagExtensions = {
```

Comment:
OK, don't know if we need to factor out a utility method just for this, I'd be OK inlining this in the ~3 places it is used.

Reply:
Yes, it seems a little redundant and has been removed